test/LoopFuserTest.cxx

```diff
@@ -318,6 +318,10 @@ FUSIBLE_DEVICE bool printAndAssign(care::host_device_ptr<int> B, int i) {
    return B[i] == 1;
 }
 
+#if defined(CARE_GPUCC)
+// parallel scans cannot work on CPU-only builds. This will call  void LoopFuser<REGISTER_COUNT,XARGS...>::flush_parallel_scans,
```

Comment:
FUSIBLE_LOOP_SCAN macro needs to work and has worked for CPU builds, for loop fuser enabled and not enabled. I suspect this is broken because you removed LoopFuser.h from the install list.